src/components/accession2/viabilityTesting/ObservationsChart.tsx

```diff
@@ -16,73 +19,103 @@ export interface Props {
 
 export default function ObservationsChart({ observations }: Props): JSX.Element {
   const classes = useStyles();
-  const chartRef = React.useRef<HTMLCanvasElement>(null);
+  const canvasRef = useRef<HTMLCanvasElement>(null);
+  const chartRef = useRef<Chart | null>(null);
+  const { loadedStringsForLocale } = useLocalization();
   const theme = useTheme();
 
-  React.useEffect(() => {
-    const ctx = chartRef?.current?.getContext('2d');
-    if (ctx) {
-      const data = observations.reverse();
-      // tslint:disable-next-line:no-unused-expression
-      new Chart(ctx, {
-        type: 'line',
-        data: {
-          labels: data.map((entry) => entry.recordingDate),
-          datasets: [
-            {
-              backgroundColor: theme.palette.accent[4],
-              data: data.map((entry) => entry.seedsGerminated),
-              fill: false,
-            },
-          ],
-        },
-        options: {
-          maintainAspectRatio: false,
-          layout: {
-            padding: {
-              left: 0,
-              right: 50,
-              top: 10,
-            },
+  useEffect(() => {
+    const createChart = async () => {
+      const ctx = canvasRef?.current?.getContext('2d');
+      if (ctx && loadedStringsForLocale) {
+        if (chartRef.current) {
+          chartRef.current?.destroy();
+          chartRef.current = null;
+        }
+
+        const localTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
Should we use the seedbank time zone here?

Reply:
Good point. This should indeed use the seed bank's time zone.

Reply:
Oh, actually, this is fine as-is, but I will add a comment to describe what it's doing since it's not very obvious (it took me a while to figure out this was needed).

Reply:
Excellent. Understood now. Thanks for the explanation!